PHP/tests/syntax_test_php.php

```diff
@@ -1750,6 +1783,37 @@ function generate2()
 ';
 // <- meta.string.php string.quoted.single.php punctuation.definition.string.end.php - meta.interpolation - string string
 
+$sql = 'SELECT ' . $col . 'FROM table WHERE ( first_name =' . $name . ')' ; . 'GROUP BY' ;
```

Comment:
I am a little curious about the purpose of this `;`.

```
$sql = 'SELECT ' . $col . 'FROM table WHERE ( first_name =' . $name . ')' ; . 'GROUP BY' ;
//                                                                        ^
```

Reply:
It is to illustrate SQL highlighting stops on expression/statement boundaries.

Reply:
We could also add tests with concated strings in parens etc.